CHANGELOG.md

```diff
@@ -64,29 +64,29 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-- [#9](https://github.com/green-code-initiative/ecoCode-php/pull/9) Upgrade licence system and licence headers of Java
+- [#9](https://github.com/green-code-initiative/creedengo-php/pull/9) Upgrade licence system and licence headers of Java
   files
-- [#10](https://github.com/green-code-initiative/ecoCode-php/pull/10) Adding EC35 rule : EC35 rule replaces EC34 with a
+- [#10](https://github.com/green-code-initiative/creedengo-php/pull/10) Adding EC35 rule : EC35 rule replaces EC34 with a
   specific use case ("file not found" specific)
-- [#13](https://github.com/green-code-initiative/ecoCode-php/issues/13) Add build number to manifest
-- [#12](https://github.com/green-code-initiative/ecoCode-php/issues/12) Fix unit tests execution with Maven
-- Update ecocode-rules-specifications to 1.4.6
+- [#13](https://github.com/green-code-initiative/creedengo-php/issues/13) Add build number to manifest
+- [#12](https://github.com/green-code-initiative/creedengo-php/issues/12) Fix unit tests execution with Maven
+- Update ecoCode-rules-specifications to 1.4.6
 - README.md upgrade : docker test environment
 
 ## [1.4.0] - 2023-08-08
 
 ### Added
 
 - PHP rules moved from `ecoCode` repository to current repository
-- [#121](https://github.com/green-code-initiative/ecoCode/issues/121) new PHP rule : Multiple if-else statement +
+- [#121](https://github.com/green-code-initiative/creedengo/issues/121) new PHP rule : Multiple if-else statement +
```

Comment:
all links with "/ecoCode/" must be update with "/creedengo-rules-specifications/" and not "/creedengo/" only